frontend/next.config.js

```diff
@@ -51,6 +52,16 @@ const moduleExports = {
    },
    async redirects() {
       return [
+         {
+            source: `/Tools/:slug(${legacyRouting.getToolRedirects()})`,
+            destination: `/Tools`,
+            permanent: false,
```

Comment:
Why wouldn't we want these to be permanent redirects?

Reply:
shoot that was for testing because if you make it permanent you have to clear the cache of the browser everytime you change the regex. ill fix that